### ISSUE TITLE
MAST for Find Markers warning

### ISSUE THREAD

**Issue by yjlee0331** [satijalab/seurat#2519]
Hi, I am trying to use MAST as the test for running the FindMarkers function.
I get the following errors: 
Warning messages:
1: In melt(coefAndCI, as.is = TRUE) :
  The melt generic in data.table has been passed a array and will attempt to redirect to the relevant reshape2 method; please note that reshape2 is deprecated, and this redirection is now deprecated as well. To continue using melt methods from reshape2 while both libraries are attached, e.g. melt.list, you can prepend the namespace like reshape2::melt(coefAndCI). In the next version, this warning will become an error.
2: In melt(lfc) :
  The melt generic in data.table has been passed a list and will attempt to redirect to the relevant reshape2 method; please note that reshape2 is deprecated, and this redirection is now deprecated as well. To continue using melt methods from reshape2 while both libraries are attached, e.g. melt.list, you can prepend the namespace like reshape2::melt(lfc). In the next version, this warning will become an error.
3: In melt(llrt) :
  The melt generic in data.table has been passed a list and will attempt to redirect to the relevant reshape2 method; please note that reshape2 is deprecated, and this redirection is now deprecated as well. To continue using melt methods from reshape2 while both libraries are attached, e.g. melt.list, you can prepend the namespace like reshape2::melt(llrt). In the next version, this warning will become an error.

I am new to R and single cell RNA seq analysis, and thus these warning messages aren't intuitive.

My code is simple -- I just put "MAST" in test.use: 
FindMarkers(Macs, ident.1 = "1", ident.2 = "2", test.use = "MAST")

Thank you!

**Comment by andrewwbutler** [comment]
Hi,

These **warnings** are from code in the MAST package and looks like they are being addressed (https://github.com/RGLab/MAST/issues/122). It should be ok to ignore them for now until the next bioconductor release.

**Comment by AMechels** [comment]
Hi, 

I also had these warnings in Seurat v3, now I updated to Seurat v4 (4.0.1) and cannot run FindMarkers with MAST anymore. I tried updating the MAST package, but that did not help. 

I receive this error before R shuts down

Done! 
Combining coefficients and standard errors

*** caught segfault ***
address 0x26e0aa18d, cause 'memory not mapped'

*** caught segfault ***
address 0xaa9, cause 'memory not mapped'
Error: no more error handlers available (recursive errors?); invoking 'abort' restart
Segmentation fault (core dumped)


I am using R version 4.0.4 (2021-02-15) -- "Lost Library Book"
Platform: x86_64-pc-linux-gnu (64-bit)

Thank you in advance!
